rts/engine/game.cc

```diff
@@ -383,6 +383,9 @@ PlayerId RTSGame::MainLoop(const std::atomic_bool *done) {
           for (const auto &bot : _bots) {
               bot->Act(_env, true);
           }
+          if (_spectator != nullptr) {
+            _spectator->Act(_env);
+          }
```

Comment:
@ppwwyyxx here is the end of game. Usually you don't need to notify the spectator unless you want to send signals. @EasyHard did you modify `_spector->Act` so that it sends signals at the end?